libs/core/src/lib/card/card-header.component.ts

```diff
@@ -37,13 +39,22 @@ export class CardHeaderComponent implements OnInit, CssClassBuilder {
     _subtitle: CardSubtitleDirective;
 
     /** @hidden */
-    constructor(private _elementRef: ElementRef<HTMLElement>) {}
+    constructor(private _elementRef: ElementRef<HTMLElement>, private renderer: Renderer2) {}
 
     /** @hidden */
     ngOnInit(): void {
         this.buildComponentCssClass();
     }
 
+    /** @hidden */
+    ngAfterContentInit(): void {
+        /** Add fd-card__avatar class to fd-avatar */
+        const avatar = this.elementRef().nativeElement.querySelector('fd-avatar');
+        if (avatar) {
+            this.renderer.addClass(avatar, 'fd-card__avatar');
+        }
+    }
```

Comment:
What about not having query selector, but `@ContentChild`? So reference to avatar class is available and `s` size can be added, when it's missing.